powerauth-java-client-jaxws/pom.xml

```diff
@@ -0,0 +1,69 @@
+<!--
+  ~ PowerAuth Server and related software components
+  ~ Copyright (C) 2017 Lime - HighTech Solutions s.r.o.
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU Affero General Public License as published
+  ~ by the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU Affero General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>powerauth-java-client-jaxws</artifactId>
+    <version>0.19.0</version>
+    <name>powerauth-java-client-jaxws</name>
+    <description>PowerAuth Server SOAP Service JaxWS Client</description>
```

Comment:
Let's use the official project name: `JAX-WS`, so the description would be `PowerAuth Server SOAP Service JAX-WS Client`.